.env.production

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_KEY="AIzaSyCO5J8pOq1hzCyeb-e5PqXa6PeGzBCV988"
```

Comment:
Seriously? 😮 

Please remove this ASAP from your code.
You should never ever put your environment variables in Github.